### PR TITLE
Israa

### PR DESCRIPTION
**Pull request by IsraaMoqbel**

### CODE REVIEW

File: src/database/db_build.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+const dbcon = require('./db_connect.js');
+const fs = require('fs');
+const sql = fs.readFileSync(`${__dirname}/select_users.js`).toString();
```
Comment (FarahZaqout) [inline src/database/db_build.js]: Shouldn't we read the db_build.sql?

Comment (FarahZaqout) [inline src/database/db_build.js]: Shouldn't we read the db_build.sql?

Comment (emanKhaledh) [reply src/database/db_build.js]: you should read from db_bulid.sql to read sql statment that build the table inside the database

File: src/database/db_build.js [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+console.log(sql);
```
Comment (emanKhaledh) [inline src/database/db_build.js]: remove log

File: src/database/db_build.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
...
+dbcon.query(sql, (err,res)=>{
+  if(err) throw new Erorr(err);
+   console.log('book_res database has been initialized with: ', res.rows);
```
Comment (emanKhaledh) [inline src/database/db_build.js]: when read file not important console.log  response

File: src/database/db_build.js (hunk 1 above)
Comment (IsraaMoqbel) [reply src/database/db_build.js]: noted! Thanks for your sharp notice!

File: src/database/db_build.sql [hunk 4]
```diff
@@ -16,12 +15,26 @@ email VARCHAR(50) UNIQUE,
...
+INSERT INTO books (name, author) VALUES
+('The Great Gatsby', 'F. Scott Fitzgerald'),
+('The Grapes of Wrath','John Steinbeck'),
+('Nineteen Eighty-Four','George Orwell'),
+('Ulysses' , 'James Joyce');
+
+INSERT INTO books (name) VALUES
+('hbrfd');
+
+select * from books;
```
Comment (ashatat) [inline src/database/db_build.sql]: I think we don't need this select line here
we can use tests to test our database

**Pull request merged**